packages/@aws-cdk/aws-ecs/lib/base/task-definition.ts

```diff
@@ -293,6 +294,40 @@ export class TaskDefinition extends TaskDefinitionBase {
     return this._executionRole;
   }
 
+  /**
+   * Validate the existence of the input target and set default values.
+   */
+  public validateTarget(options: LoadBalancerTargetOptions): LoadBalancerTarget {
```

Comment:
I think I might actually be fine with making these `@internal` and prefixing them with `_` after all.

I don't really want to expose them to construct users.